packages/venia-concept/src/components/PurchaseDetailsPage/OrderItemsList/__tests__/orderItemsList.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { List } from '@magento/peregrine';
+import OrderItemsList from '../OrderItemsList';
+import OrderItem from '../../OrderItem';
+
+configure({ adapter: new Adapter() });
+
+const classes = {
+    header: 'header'
+};
+
+const itemsMock = [
+    {
+        id: 1,
+        name: 'name',
+        size: '42',
+        color: 'color',
+        qty: 1,
+        titleImageSrc: 'picture',
+        price: 27,
+        sku: 'sku'
+    },
+    {
+        id: 1,
+        name: 'name',
+        size: '42',
+        color: 'color',
+        qty: 1,
+        titleImageSrc: 'picture',
+        price: 27,
+        sku: 'sku'
+    },
+    {
+        id: 1,
+        name: 'name',
+        size: '42',
+        color: 'color',
+        qty: 1,
+        titleImageSrc: 'picture',
+        price: 27,
+        sku: 'sku'
+    }
+];
+
+test('renders correctly', () => {
+    const wrapper = shallow(
+        <OrderItemsList classes={classes} title="title" items={itemsMock} />
+    ).dive();
+
+    expect(wrapper.find('.header')).toHaveLength(1);
+    expect(wrapper.find(List)).toHaveLength(1);
+    expect(
+        wrapper
+            .find(List)
+            .children()
+            .find(OrderItem)
```

Comment:
```
wrapper.find(OrderItem)
```

And here it is better to test at least that we rendered all order items from items array.